webview_panels/src/modules/app/appSlice.ts

```diff
@@ -1,15 +1,15 @@
 import { createSlice, PayloadAction } from "@reduxjs/toolkit";
-import { AppStateProps } from "./types";
+import { AppStateProps, Themes } from "./types";
 
 export const initialState = {
-  theme: "",
+  theme: Themes.Dark,
```

Comment:
@saravmajestic is this intentional?

Reply:
@anandgupta42 yes, i have to apply theme for the markdown based on vscode theme. This initial value value will be overwritten after the page is loaded with right value